src/App.tsx

```diff
@@ -16,35 +19,69 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const handleReset = () => {
+    setSortField(SortType.DEFAULT);
+    setIsReversed(false);
+  };
```

Comment:
The function's name `handleReset` is appropriate as it starts with a verb, but consider renaming it to more precisely reflect its action, such as `resetSortAndOrder` to make it clearer what the method does. [CODE STYLE]